src/overlays/actors/ovl_En_Sb/z_en_sb.c

```diff
@@ -21,44 +35,363 @@ const ActorInit En_Sb_InitVars = {
     (ActorFunc)EnSb_Update,
     (ActorFunc)EnSb_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Sb_0x808E3EF0/EnSb_Init.asm")
+static ColliderCylinderInitType1 sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_ENEMY,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0xF7CFFFFF, 0x04, 0x08 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
+    { 30, 40, 0, { 0, 0, 0 } },
+};
+
+static DamageTable sDamageTable = {
+    0x00, 0x00, 0x00, 0xF1, 0xF1, 0xF1, 0x00, 0xF1, 0x00, 0x00, 0x00, 0xF1, 0xF1, 0xF1, 0x00, 0x00,
+    0xF1, 0x00, 0x00, 0xF1, 0x00, 0x00, 0x00, 0xF1, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xF1,
+};
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_S8(hintId, 39, ICHAIN_CONTINUE),
+    ICHAIN_U8(targetMode, 2, ICHAIN_CONTINUE),
+    ICHAIN_F32(targetArrowOffset, 30, ICHAIN_STOP),
+};
+
+static Vec3f sFlamePosOffsets[] = {
+    { 5.0f, 0.0f, 0.0f },
+    { -5.0f, 0.0f, 0.0f },
+    { 0.0f, 0.0f, 5.0f },
+    { 0.0f, 0.0f, -5.0f },
+};
+
+extern FlexSkeletonHeader D_06002BF0;
+extern AnimationHeader D_06000194;
+extern AnimationHeader D_0600004C;
+extern AnimationHeader D_06000124;
+extern AnimationHeader D_06002C8C;
+extern AnimationHeader D_060000B4;
+
+void EnSb_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnSb* this = THIS;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+    this->actor.colChkInfo.damageTable = &sDamageTable;
+    this->actor.colChkInfo.mass = 0xA;
+    this->actor.colChkInfo.health = 2;
+    SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_06002BF0, &D_06000194, this->limbDrawTable,
+                     this->transitionDrawTable, 9);
+    Collider_InitCylinder(globalCtx, &this->collider);
+    Collider_SetCylinderType1(globalCtx, &this->collider, &this->actor, &sCylinderInit);
+    this->isDead = false;
+    this->actor.colChkInfo.mass = 0x5A;
```

Comment:
Oh, this actor sets its mass twice for some reason. Anybody knows if this has some side effect?
I think you should at least put a comment about this here.